src/components/layout/layout-v2.tsx

```diff
@@ -35,68 +41,32 @@ const Main = styled.main`
    */
   display: grid;
   grid-template-columns:
-    1fr
-    min(821px, calc(100% - 2 * 24px))
-    1fr;
+    [main-start] minmax(24px, 1fr)
+    [content-start] minmax(0, 820px) [content-end]
+    minmax(24px, 1fr) [main-end];
```

Comment:
@feedm3 I have refactored this one to let the grid do the calculations. I developed that within a hero and though it's worth to replace in the layout (admittedly not much easier though)

In addition created named tracks to assign the content to names and not numbers (`grid-column: content`)